source/extensions/filters/http/ext_proc/ext_proc.cc

```diff
@@ -660,6 +659,15 @@ void Filter::onReceiveMessage(std::unique_ptr<ProcessingResponse>&& r) {
     break;
   }
 
+  if (!logging_info_upstream_host_set_ && stream_) {
```

Comment:
You don't really need a new flag here. if `upstream_host_` is not nullptr, it means it has been set.

Reply:
Done.